src/azure-cli/azure/cli/command_modules/netappfiles/tests/latest/test_volume_commands.py

```diff
@@ -71,6 +70,33 @@ def test_create_delete_volumes(self):
         volume_list = self.cmd("netappfiles volume list --resource-group {rg} -a %s -p %s" % (account_name, pool_name)).get_output_in_json()
         assert len(volume_list) == 0
 
+    @ResourceGroupPreparer(name_prefix='cli_tests_rg')
```

Comment:
Can you use a more descriptive prefix for your resource group? e.g. clitestnetappfilesvolume

Reply:
ok will update